internal/repository/repository.go

```diff
@@ -17,7 +17,13 @@ var (
 	ErrNoSuchRequest     = errors.New("request with the given id does not exists")
 )
 
-const uniqueViolationCode = "23505"
+const (
+	uniqueViolationCode = "23505"
+
+	RequestStatusProcessed = "processed"
+	RequestStatusFailed    = "failed"
+	RequestStatusDone      = "done"
```

Comment:
What is the difference between `processed` and `done`?

Reply:
> What is the difference between `processed` and `done`?

This was written in the Technical Task :) 
Maybe this should show that the converter has already started processing and we could track the stage of readiness of the request.

Reply:
Then let's call it `processing` or `in_progress`